adagrams/game.py

```diff
@@ -1,11 +1,74 @@
-def draw_letters():
-    pass
+import random
+letter_set1 = ["A","I"]* 9
+letter_set2 = ["B","C","F","H","M","P","V","W","Y"] * 2
+letter_set3 = ["D","L","S","U"] * 4
+letter_set4 = ["E"] * 12
+letter_set5 = ["G"] * 3
+letter_set6 = ["J","K","Q","X","Z"]
+letter_set7 = ["N","R","T"] * 6
+letter_set8 = ["O"]* 8
+letters_list = letter_set1 + letter_set2 + letter_set3 + letter_set4\
+               + letter_set5 + letter_set6 + letter_set7 + letter_set8
```

Comment:
I see how you're building up `letter_lists` so that the pool of letters a player can draw from has an accurate distribution of letters. I suggest a different approach though. Lines 2-10 is a lot of different global variables and when possible, we should try to reduce the number of global variables if we can just use one.  

What if you create a constant variable called LETTER_POOL that is a dict like this:
```python
LETTER_POOL = {
    'A': 9, 
    'B': 2, 
    'C': 2, 
    'D': 4, 
    'E': 12, 
    # etc...
}
```

It is a long-ish variable but it's easy to understand what it is and what the keys and values mean. Then you can iterate over the key value pairs with a simple for loop to create letters_list. I prefer this approach because it's more readable and easier to understand what a for loop would be doing. It also means we only have a single variable LETTER_POOL versus having many variables like letter_set1, letter_set2, etc. 

A last thought ere about your global variables: `letter_setX` is a little misleading because set is a datatype in Python so we really have a list, but someone reading your code might think we're dealing with sets. We should pay attention to the words we use when naming variables because sometimes our variable name needs to describe the datatype.